### ISSUE TITLE
devon release: unable to determine version if exec-plugin is used by project

### ISSUE THREAD

**Issue by hohwille** [devonfw/devon-ide#133]
```
➜  code git:(master) devon release
You are inside a top-level maven project
Trying to determine current version of your project...

******** ATTENTION ********
ERROR: Failed to determine version. You need to run 'mvn install' before.

******** ATTENTION ********
ERROR: Multiple versions in reactor:
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.apache.commons.exec.DefaultExecutor.execute (DefaultExecutor.java:166)
    at org.apache.commons.exec.DefaultExecutor.executeInternal (DefaultExecutor.java:404)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:956)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.codehaus.mojo.exec.ExecMojo.execute (ExecMojo.java:313)
    at org.codehaus.mojo.exec.ExecMojo.executeCommandLine (ExecMojo.java:751)
    at org.codehaus.mojo.exec.ExecMojo.executeCommandLine (ExecMojo.java:804)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
1.0.0-beta2-SNAPSHOT
[ERROR]
[ERROR]   mvn <goals> -rf :mmm-code-java-impl
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR] Command execution failed.
[ERROR] Failed to execute goal org.codehaus.mojo:exec-maven-plugin:1.6.0:exec (default-cli) on project mmm-code-java-impl: Command execution failed.: Process exited with an error: 1 (Exit value: 1) -> [Help 1]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Unknown lifecycle phase "1.0.0-beta2-SNAPSHOT". You must specify a valid lifecycle phase or a goal in the format <plugin-prefix>:<goal> or <plugin-group-id>:<plugin-artifact-id>[:<plugin-version>]:<goal>. Available lifecycle phases are: validate, initialize, generate-sources, process-sources, generate-resources, process-resources, compile, process-classes, generate-test-sources, process-test-sources, generate-test-resources, process-test-resources, test-compile, process-test-classes, test, prepare-package, package, pre-integration-test, integration-test, post-integration-test, verify, install, deploy, pre-clean, clean, post-clean, pre-site, site, post-site, site-deploy. -> [Help 1]
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/LifecyclePhaseNotFoundException
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] --------------------------------[ pom ]---------------------------------
[INFO] -----------------------< testing:maven.project >------------------------
[INFO] BUILD FAILURE
[INFO] Building Test_Local_Maven_Project 0.0.1-SNAPSHOT                   [1/3]
[INFO] Finished at: 2019-07-02T14:12:58+02:00
[INFO] Reactor Build Order:
[INFO] Reactor Summary for Test_Local_Maven_Project 0.0.1-SNAPSHOT:
[INFO] Scanning for projects...
[INFO] Test_Local_Maven_Project                                           [jar]
[INFO] Test_Local_Maven_Project                                           [pom]
[INFO] Test_Local_Maven_Project ........................... FAILURE [  0.002 s]
[INFO] Test_Local_Maven_Project ........................... SKIPPED
[INFO] Total time:  0.262 s
org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1)
We are sorry for the inconvenience. Please check the above errors, resolve them and try again.
ERROR: command 'build -q get-version' failed with exit code 255
We are sorry for the inconvenience. Please check the above errors, resolve them and try again.
```

**Comment by hohwille** [comment]
https://groups.google.com/forum/#!topic/mojohaus-dev/gBVMkAbvrfE

**Comment by hohwille** [comment]
Fixed with PR #134
